rest/model/waterfall.go

```diff
@@ -0,0 +1,36 @@
+package model
+
+import (
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/pkg/errors"
+)
+
+type APIWaterfallData struct {
+	Rows              []model.WaterfallRow     `json:"rows"`
+	Versions          []model.WaterfallVersion `json:"versions"`
```

Comment:
the public api structures can't pass db model data straight through.

Reply:
Why not?

Reply:
This isn't a regular model at all.